geometry_msgs/PoseStamped place_position
---
bool result
---
std_msgs/Float32 progress